tasks.sql

```diff
@@ -0,0 +1,119 @@
+﻿USE [Company]
+GO
+
+/**1) Получить все содержимое таблицы Employee*/
+SELECT * FROM Employee
+GO
+
+
+/*2) Получить коды и названия должностей, чья минимальная зарплата не превышает 500*/
+SELECT Id, Position FROM Job 
+WHERE MinimalSalaryPerMonth <= 500
+GO
+
+
+/*3) Получить среднюю заработную плату начисленную в январе 2015 года*/
+SELECT AVG (Amount) FROM Salary 
+WHERE Year = 2015 and Month = 1
+GO
+
+
+/*4) Получить имя самого старого работника, а также его возраст*/
+SELECT Name FROM Employee WHERE Birthday = (
+	SELECT MIN(Birthday) FROM Employee
+	);
+GO
+
+
+/*5) Найти фамилии работников, которым была начислена зарплата в январе 2015 года*/
+SELECT Name FROM Employee
+WHERE Id IN (
+	SELECT EmployeeId FROM Salary 
+	WHERE Year = 2015 and Month = 1
+)
+GO
+
+
+/*6) Найти коды работников, зарплата которых в мае 2015 года снизилась по сравнению с 
+каким-либо предыдущим месяцем этого же года*/
+SELECT EmployeeId
+FROM
+(
+	SELECT EmployeeId, MAX(AMOUNT) as innerMax
+	FROM Salary
+	WHERE Year = 2015 and Month < 5
+	GROUP BY EmployeeId
+) inner_query
+WHERE EmployeeId IN (
+	SELECT EmployeeId 
+	FROM Salary
+	WHERE Year = 2015 and Month = 5 and Amount < innerMax
+)
+GO
+
+
+/*7) Получить информацию о кодах, названиях отделов и количестве работающих в этих 
+	отделах в настоящее время сотрудников*/
+SELECT Id, Name, 
+	(SELECT COUNT(EmployeeId) 
+	 FROM Career 
+	 WHERE DismissalDate IS NULL and DepartmentId = d.Id
+	)
+FROM Department AS d
+
+
+/*8) Найти среднюю начисленную зарплату за 2015 год в разрезе работников*/
+SELECT AVG(innerAverages)
+FROM
+(
+	SELECT AVG(AMOUNT) as innerAverages
+	FROM Salary AS s
+	WHERE Year = 2015 
+	GROUP BY EmployeeId
+) inner_query
+GO
+
+
+/*9) Найти среднюю зарплату за 2015 год в разрезе работников. Включать в результат только тех работников, 
+	начисления которым проводились не менее двух раз*/
+SELECT AVG(innerAverages)
+FROM
+(
+	SELECT AVG(AMOUNT) as innerAverages
+	FROM Salary AS s
+	WHERE Year = 2015 
+	GROUP BY EmployeeId
+	HAVING COUNT(Id) > 2
+) inner_query
+GO
+
+
+/*10) Найти имена тех работников, начисленная зарплата которых за январь 2015 превысила 1000*/
+SELECT Name FROM Employee AS e
```

Comment:
I guess, it is more readable when we write "FROM" on a new line:

SELECT someField1, someFfiled2
FROM someTable;